.github/workflows/tests.yaml

```diff
@@ -34,6 +44,11 @@ jobs:
           poetry install
 
       - name: Run pytest
+        env:
```

Comment:
Note: this is required for tests, since the test code will import target app, and the app will need to load credentials for MySQL, when we have test case with database transactions.